cfgov/cfgov/settings/base.py

```diff
@@ -300,6 +300,13 @@
     d for d in REPOSITORY_ROOT.joinpath("static.in").iterdir() if d.is_dir()
 ]
 
+# Collect static files into, and serve them from, cfgov/collectstatic,
+# unless otherwise specified via DJANGO_STATIC_ROOT.
+STATIC_ROOT = os.getenv("DJANGO_STATIC_ROOT", REPOSITORY_ROOT / "collectstatic")
+
+# Serve files under cfgov/root at the root of the website.
+WHITENOISE_ROOT = PROJECT_ROOT / "root"
```

Comment:
I'm open to other suggestions for where these files could live, besides creating a new `cfgov/root`. Leaving them in `cfgov/unprocessed/root` isn't right because they no longer get processed by the frontend build. I could put them into some app's static files (like `cfgov/core/static/root`) but then we couldn't use `WHITENOISE_ROOT` to serve them and would need to write some manual Django URL handling to serve them.